rewrite-core/src/main/java/org/openrewrite/marker/GitProvenance.java

```diff
@@ -39,8 +39,7 @@
 
 @Value
 @With
-public class GitProvenance implements Marker {
-    UUID id;
+public class GitProvenance {
```

Comment:
As I mentioned offline, this will cause existing ASTs which have `GitProvenance` markers on them to fail to deserialize. Could largely work around the problem by not removing the `Marker` interface here.